GDJS/Runtime/types/project-data.d.ts

```diff
@@ -88,6 +88,11 @@ export interface LayerData {
   visibility: boolean;
   cameras: CameraData[];
   effects: EffectData[];
+  r: number;
```

Comment:
nit: see my other comment, let's give these more specific names, and rename "syncWithBaseLayer" to "followBaseLayerCamera"